### PR TITLE
Implemented withdraw method

### PR DESCRIPTION
**Pull request by vlad-sundunchan**

### CODE REVIEW

**Comment by markblaize** [comment]
Task linked: [CU-344xbab Implement withdraw method](https://app.clickup.com/t/344xbab)

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,7 @@ contracts/leverage_trading/target
 /node_modules/
 /res/
 /.near-credentials
+contracts/leverage_trading/res/*.wasm
```
Comment (TymofiiSavinov) [inline .gitignore:14]: change it contracts/leverage_trading/res/*.wasm => contracts/leverage_trading/res/

File: build.sh [hunk 2]
```diff
@@ -2,5 +2,10 @@
 set -e
 
 RUSTFLAGS='-C link-arg=-s' cargo build --manifest-path ./contracts/Cargo.toml --target wasm32-unknown-unknown --release
+RUSTFLAGS='-C link-arg=-s' cargo build --manifest-path ./contracts/leverage_trading/Cargo.toml --target wasm32-unknown-unknown --release
```
Comment (TymofiiSavinov) [inline build.sh:5]: remove changes in build.sh, omomo build should not contain leverage_trading

File: test.sh [hunk 3]
```diff
@@ -2,3 +2,5 @@
 set -e
 
 cargo test --manifest-path ./contracts/Cargo.toml -- --nocapture
+cargo test --manifest-path ./contracts/leverage_trading/Cargo.toml -- --nocapture
```
Comment (TymofiiSavinov) [inline test.sh:5]: same

File: .gitignore (hunk 1 above)
Comment (vlad-sundunchan) [reply .gitignore]: Changed it to `contracts/leverage_trading/res/*`. Because the `contracts/leverage_trading/res/` ignores the folder

**Review by TymofiiSavinov — DISMISSED** [review]

**Review by TymofiiSavinov — APPROVED** [review]

**Pull request merged**